examples/keycloak/server.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs')
+const path = require('path')
+const express = require('express')
+const { makeExecutableSchema } = require('graphql-tools')
+
+const { ApolloVoyagerServer, gql } = require('../../packages/apollo-voyager-server')
+const { KeycloakSecurityService } = require('../../packages/apollo-voyager-keycloak')
+
+
+const keycloakConfig = JSON.parse(fs.readFileSync(path.resolve(__dirname, './config/keycloak.json')))
+
+// This is our Schema Definition Language (SDL)
+const typeDefs = gql`
+
+  # In the older version of graphql in the data sync server
```

Comment:
WOW. This means that users will need to define our directives?
Maybe we could concatenate user defined schema with our directives later.
Need to track this change upstream because I cannot believe that this is required. 
In previous version makeExecutableSchema accepted separate directives field.

Reply:
`makeExecutableSchema` still accepts the separate directives field however, in `graphql` there is a function called `buildASTSchema` (which is used by `makeExecutableSchema`). This function does not take that directives field into account when validating the input typedefs. In the older version of `graphql`, there was way less validation.

It's possible that the old way was actually invalid and that this new behaviour is the intended behaviour. It might be worth opening an issue against `graphql-tools` because Apollo still have some example code in their docs using the old way.

Reply:
We could ask them if they know about this change. Is it the intended behaviour etc.

Reply:
Just noticed that they examples aren't using latest version of graphql. Maybe they do that intentionally and there is recommended version set from apollo team. Anyway good to ask around